### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by hellonadya**
- add v6.2 roadmap

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -5,6 +5,51 @@ This document provides the roadmap of the planned ONLYOFFICE Document Server cha
 This is an updated and corrected version of the roadmap.
 We also reserve the right to change it when necessary.
 
+## Version 6.2 
+
+### All editors
+
+* Setting font size manually
```
Comment (ShockwaveNN) [inline ROADMAP.md:12]: Not clear what it is
We already got one
image

File: ROADMAP.md [hunk 2]
```diff
@@ -5,6 +5,51 @@ This document provides the roadmap of the planned ONLYOFFICE Document Server cha
...
+* New options to work with text fields
+* Table of figures
+
+### Spreadsheet editor
+
+* Autoexpansion for formatted tables
+* Slicers for pivot tables
+* Data validation settings for cells
+* Custom number format
+* New functions: growth, trend, logest, unique
```
Comment (ShockwaveNN) [inline ROADMAP.md:27]: We used to name functions in captal letter, like `GROWTH`, `TREND` etc

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 2 above)
Comment (hellonadya) [reply ROADMAP.md]: fixed

File: ROADMAP.md (hunk 1 above)
Comment (hellonadya) [reply ROADMAP.md]: Setting font size up to 300px (up to 409px in spreadsheets) manually

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**